FAQ.md

```diff
@@ -1,202 +1,122 @@
 
-> My point is: 
-> 
-> The language (or framework) doesn't matter, learning how to understand a problem and work through it. There's concepts like inheritance composition, higher order functions, OOP, FP, abstraction, module pattern, pub/sub, parallel code,  and more that you can't learn just from writing code. This stuff is some really big conceptual stuff in CS. To learn those, you need a good teacher or course. 
-> 
-> When to apply a pattern or algorithm to a problem isn't something a language itself can teach you.
-> 
-> also, read this: https://medium.com/@bycdiaz/choosing-the-right-language-a-short-guide-on-how-not-to-ruin-your-career-2b353be1371
+**Should I take notes?** 
 
-**TOP doesn't offer xyz language, what should I do?**
+Mostly, no. Understanding > memorization. Read more information [here](https://discord.com/channels/505093832157691914/505093832157691916/768161823366578176).
 
-Read the above answer's link. It covers this.
+**Do I need a degree to get a job?**
 
-**In what order should I take the courses of the curriculum?**
+Generally no, however a degree can help you get past some non-understanding HR people. Many people have become successful without them, see <#1013804065169346631>.
 
-You should start at the beginning and not skip, even if you feel you are experienced. Part of what makes TOP work is building on previous knowledge in the curriculum. If you know the stuff, it will go quickly, if you do not, you'll be glad you did it. TOP expects you to have gone through the previous sections before each next section. This is a curriculum, not a collection of tutorials.
+**Should I do multiple courses at the same time?** 
 
-**Should I take notes?**
+No. [See this for more information](https://discord.com/channels/505093832157691914/505093832157691916/778727680438698055). 
 
-Mostly, no. Understanding > memorization. 
+**Is there an Odin dark theme?** 
 
-**Do I need a degree to get a job?**
+Yup! If you are not logged in, you will see a little moon and sun icon in the top nav bar. If you are logged in, you will see the choice when you click on the user icon in the top nav bar. Keep in mind it is in beta, so if you see something that needs to be addressed, you can let us know in the suggestions-bugs channel. 
 
-Generally no, however a degree can help you get past some non-understanding HR people. Many people have become successful without them, see #odin-success-stories.
+**Is TOP hard?** 
 
-**Should I do multiple courses at the same time?**
-No, 
+Very, but that doesn't mean you can't do it. Learning programming in general is tough, no matter where you go. Grit is something you will need to develop to get through it, read about it [here](https://en.wikipedia.org/wiki/Grit_(personality_trait)). 
```

Comment:
I also considered linking to one of the articles shared in the 'motivation and mindset' lesson instead of a Wikipedia article. Maybe the video even but I wasn't sure. What do you think?